vllm/v1/executor/multiproc_executor.py

```diff
@@ -38,6 +39,19 @@ def __init__(self, vllm_config: VllmConfig) -> None:
         # and ensure workers will be terminated.
         self._finalizer = weakref.finalize(self, self.shutdown)
 
+        # The child processes will send SIGQUIT when unrecoverable
+        # errors happen.
+        def sigquit_handler(signum, frame):
+            logger.fatal(
+                "MulitprocExecutor got SIGQUIT from worker processes, shutting "
+                "down. See stack trace above for root cause issue.")
+            # Propagate error up to parent process.
+            parent_process = psutil.Process().parent()
+            parent_process.send_signal(signal.SIGQUIT)
+            self.shutdown()
+
+        signal.signal(signal.SIGQUIT, sigquit_handler)
```

Comment:
BTW: why use `SIGQUIT` for these? Users might get weird error messages if they hit `ctrl-\`, so I was thinking it might be better to use `SIGUSR1`

Reply:
Mostly because this was inspired by SGL and they use `SIGQUIT`. I will switch to `SIGUSR1`